routing/cross_mwm_index_graph.hpp

```diff
@@ -124,15 +150,38 @@ class CrossMwmIndexGraph final
     return true;
   }
 
-  RoutingIndexManager & m_indexManager;
+  template <class Fn>
+  CrossMwmConnector const & Deserialize(NumMwmId numMwmId, Fn && fn)
+  {
+    std::unique_ptr<FilesContainerR::TReader> reader =
+        make_unique<FilesContainerR::TReader>(GetValue(numMwmId).m_cont.GetReader(CROSS_MWM_FILE_TAG));
```

Comment:
Пока нет. У нас 11 стандарт. В нем нет std::make_unique. Используем нашу из std/unique_ptr
```
/// @todo(y): replace this hand-written helper function by
/// std::make_unique when it will be available in C++14
template <typename T, typename... Args>
unique_ptr<T> make_unique(Args &&... args)
{
  return unique_ptr<T>(new T(std::forward<Args>(args)...));
}
```